operations/configuration_management/Target_Ansible_Tasks_for_Image_Customization.md

```diff
@@ -3,29 +3,75 @@
 The Configuration Framework Service \(CFS\) enables Ansible playbooks to run against both running nodes \(node personalization\) and images prior to boot\(image customization\).
 See [Configuration Management Use Cases](Configuration_Management.md#use-cases) for more information about image customization and when it should be used.
 
-Ideally image customization playbooks should be separate from node personalization playbooks. This reduces the likelihood that Ansible content will run in both modes, and reduces the need for conditional checks in the playbooks.
+## Using `cray_cfs_image`
 
 For situations when both image customization and node personalization need to be part of the same playbook, CFS provides the `cray_cfs_image` variable to distinguish between
 node personalization and image customization. When this variable is set to `true`, it indicates that the CFS session is an image customization and the playbook is targeting an image.
 
-## Using `cray_cfs_image`
+It is best practice to include a default in Ansible roles for playbook and role portability because CFS injects this variable at runtime. This can be done in the defaults section of the role, or where the variable is called. For example:
 
-`cray_cfs_image` can be set with [Ansible playbook conditionals](https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html)
-to selectively run individual tasks with `when: cray_cfs_image`, or to ignore individual tasks with `when: not cray_cfs_image`.
+```text
+{{ cray_cfs_image | default(false) }}
```

Comment:
I don't know for sure, but I suspect this isn't a problem because it's stock go templating.  I don't expect we're falling back on this though.  We inject this variable into the environment, so this is more of a safety/good practice thing.